e2e/charts/newrelic-infrastructure-k8s-e2e/templates/daemonset.yaml

```diff
@@ -54,12 +54,10 @@ spec:
             {{- end }}
           env:
             {{- if eq .Values.daemonset.clusterFlavor "Minikube" }}
```

Comment:
I'd take the advantage of your enlightment about what these checks do and add a small comment describing them.

Reply:
Sure, added some.